### DISCUSSION TITLE
Small change to where ads are

### DISCUSSION THREAD

**Discussion by frogfrogfrogfrogfrog** [monkeytypegame/monkeytype#930] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/58377021/107097077-5e1d5480-67c9-11eb-8303-60bb736daa31.png)

It would be nice if the ads were spaced out 3, 3, 3 instead of 2, 4, 3

**Comment by Miodec** [comment, marked as answer]
Alright, will fix that.
